javalin/src/main/java/io/javalin/core/util/Headers.kt

```diff
@@ -0,0 +1,144 @@
+/*
+ * Javalin - https://javalin.io
+ * Copyright 2021 David Åse
+ * Licensed under Apache 2.0: https://github.com/tipsy/javalin/blob/master/LICENSE
+ */
+
+package io.javalin.core.util
+
+import java.time.Duration
+import java.util.Arrays
+import java.util.Locale
+import java.util.stream.Collectors
+import kotlin.collections.HashMap
+
+/**
+ * A plugin to configure arbitrary headers, with a focus on the OWASP secure headers project
+ * https://owasp.org/www-project-secure-headers/
+ */
+class Headers {
+
+    val headers: MutableMap<String, String> = HashMap(10)
+
+    // Strict-Transport-Security: max-age=31536000 ; includeSubDomains
+    fun strictTransportSecurity(duration: Duration, includeSubdomains: Boolean) {
```

Comment:
This could be 

```kotlin
fun strictTransportSecurity(duration: Duration, includeSubdomains: Boolean) {
    headers[Header.STRICT_TRANSPORT_SECURITY] = 
        "max-age=${duration.seconds}" + if (includeSubDomains) " ;true" else ""
}
```

Is that the syntax? Either you append ` " ;true"`  or nothing?

Reply:
you append "includeSubDomain".. did that with some refactoring